src/pages/Home/Results/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState, useEffect, useMemo } from "react";
+import Group from './Group';
+import ContentFrame from '../../../components/ContentFrame';
+
+type ResultsProps = {
+  filterText: string;
+}
+
+function Home({ filterText }: ResultsProps) {
+  const [groups, setGroups] = useState<OpenVE.Group[]>([]);
+  const [icons, setIcons] = useState<OpenVE.LangIcons>({});
+  
+  useEffect(() => {
+    const fetchData = async () => {
+      const data = await fetch('https://raw.githubusercontent.com/estebanborai/openve-openve.github.com/migrate/react/static/telegram-groups.json');
```

Comment:
I guess this url can be made configurable right? (As in pass the value via Environment variable when deploying)

Reply:
I don't think that is required to have this as an environment variable because this URL will be static once we have defined our groups source. So, I could change it now but I think that it doesn't makes sense atm because we are missing this source now.

Reply:
Yeah, and once we have it, code changes will be required to be able to change it again. hardcoded input data, is an antipattern... While it what I'm saying could be considered premature optimization, it's something quicker to be dealt with (After all, you have more knowledge of react apps than I do) than `solving` it in the future

Reply:
Yep now that you said it in that way yes.
I used a hardcoded value because is not dynamic, it doesnt makes sense to use an environment variable for a constant.

Anyway, the bad thing here is that this url is pointing to my pr, thats not okay. I leave it like that because we dont have any other resource for this data, I think this problem is related to the same as the search, maybe the data structure is not the one we need and we need to define the scope of the organization, as far as I know it was technology but as you said above we also have sports here, so maybe we need another structure for our data.

## Suggestion

What if we define the source for the groups, design the right data structure and having that done we get back to this?

Reply:
We've got a data structure to begin with and we _know_ where a source would be, redefining the right data structure can be done afterwards. This doesn't stops you from making it configurable right? :)

Reply:
Sure! This can be configurable using an environment variable at build time through GitHub Actions.

I need to know where is going to be this configuration, and access to the data used so we have this up and running when deploying to prod.